client/src/pages/RoomPage/index.jsx

```diff
@@ -34,15 +34,16 @@ export const RoomPage = ({ user, socket }) => {
   useEffect(() => {
     // add users in a array(usersInRoom)
     socket.on("userIsJoined", ({ users }) => {
-      console.log(users);
       setUsersInRoom(users);
```

Comment:
users suppose to be an array of users present in a perticular room, but it is not an array, it sets the current user to the state "usersInRoom" !!!